kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStepDialog.java

```diff
@@ -546,6 +607,47 @@ private void mapAllInputs() {
     }
   }
 
+  /**
+   * Parses an input String to CS naming conventions for table and column names
+   * @param input, input String
+   * @return parsed output String
+   */
+  private String parseTableColumnNameToCSConvention(String input){
+    StringBuilder output = new StringBuilder();
+
+    if(input == null){
+      output.append("null");
+    }else{
+      //if the first character is lowercase [a-z] use it
```

Comment:
As discussed the limitation is documented poorly, the first character needs to be a letter but it can be upper or lower case so don't restrict to just lower case.